src/components/DebounceInput/DebounceInput.tsx

```diff
@@ -20,7 +20,7 @@ export const DebounceInput = (props: DebounceInputProps) => {
 
   return (
     <Input
-      css={null}
+      css={undefined}
```

Comment:
could `css` be removed, or does it need to be explicitly set?

Reply:
Yeah, it does need to be explicitly set. :( At least it has in the past.

Reply:
I think now you can completely remove the `css` property. That is no longer required .

Reply:
> null is no longer valid

when did the change happen? i'm mostly just wondering if this is an 8.x change or 7.x change since i think most of the data sources are pointing to the main branch of this repo

Reply:
I think it is changed from 7.5 https://github.com/grafana/grafana/pull/31479